docs/src/kpaths.md

```diff
@@ -85,4 +85,32 @@ Main.HTMLPlot(P¹², 525) # hide
 plot(::KPathInterpolant, ::Any, ::Layout)
 ```
 
+## K-paths for non-standard lattices
+One can create a **k**-path for a non-standard unit cell by using Spglib.
+```@example kpath
+using Spglib
+# Trigonal lattice, space group R-3m, 166
+a = 1.0
+c = 8.0
+lattice_standard    = SVector(SVector(a*sqrt(3)/2,  a/2, c/3),
+                              SVector(-a*sqrt(3)/2, a/2, c/3),
+                              SVector(0.0, -a, c/3))
+lattice_nonstandard = SVector(SVector(a*sqrt(3)/2, -a/2, c/3),
+                              SVector(0.0, a, c/3),
+                              SVector(-a*sqrt(3)/2, -a/2, c/3))
```

Comment:
```suggestion
pRs_standard    = [[a*√3/2,  a/2, c/3],
                   [-a*√3/2, a/2, c/3],
                   [0, -a, c/3]]
pRs_nonstandard = [[a*√3/2, -a/2, c/3],
                   [0, a, c/3],
                   [-a*√3/2, -a/2, c/3]]
```
We don't have StaticArrays loaded in this example I think (and we don't need it technically).
Also changed from `lattice` to `pRs` to sync with the notation elsewhere.